tests/defaults.yml

```diff
@@ -1,3 +1,33 @@
-# Overwrite parameters here
-
-# parameters: {...}
+parameters:
+  group_sync_operator:
+    sync:
+      foo:
+        schedule: '* * * * *'
+        providers:
+          keycloak:
+            credentials:
+              stingData:
+                username: foo
+                password: bar
```

Comment:
For new stuff we should manage secrets outside of the provider-like structures.
* See https://github.com/appuio/component-openshift4-authentication/pull/28#pullrequestreview-735844858
* and an implementation in https://github.com/appuio/component-openshift4-authentication/pull/32
Basically the structure becomes
```yaml
parameters:
  group_sync_operator:
    sync:
      foo:
        providers:
          keycloak:
            secretName: foo-keycloak
    secrets:
      foo-keycloak:
        password: something
```
That way, secrets can be better integrated into Vault, but also reused for different providers.

Reply:
Good to know. I missed that decision.

Reply:
well, it's not so officially communicated either :) But we did start a best-practice page